src/pages/auth/LoginPage.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <q-page>
+  <q-page>npm
```

Comment:
typo as has been said
```suggestion
  <q-page>
```